fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1411,13 +1411,13 @@ ACTOR Future<GetReadVersionReply> getLiveCommittedVersion(ProxyCommitData* commi
 	rep.version = commitData->committedVersion.get();
```

Comment:
won't these assignments be override in the following for() loop for the same reason? 
Why don't we move them below the for loop?

Reply:
I think these are fine because they are supposed to overridden when other proxy has a larger version and it's 'locked' and 'metadataVersion' is more up-to-date.